src/utils/util.ts

```diff
@@ -189,7 +189,7 @@ export const readPackageJson = async (config: d.ValidatedConfig, compilerCtx: d.
 export type ParsePackageJsonResult = {
   diagnostic: d.Diagnostic | null;
   data: any | null;
-  filePath: string;
+  filePath: string | undefined;
```

Comment:
See my other comment in this file - if we can remove the changes proposed there, I _think_ we won't need this either.